hw/Makefile

```diff
@@ -105,13 +107,21 @@ $(ips_reg): %_reg: | $(REG_OUTPUT_DV_DIR)
 .PHONY: otp-mmap
 $(filter otp_ctrl_reg,$(ips_reg)): otp-mmap
 otp-mmap:
-	cd ${PRJ_DIR} && ./util/design/gen-otp-mmap.py
+  ifeq ($(USE_BUFFER),1)
+		cd ${PRJ_DIR} && ./util/design/gen-otp-mmap.py --entropy-buffer ./util/design/otp_mmap_entropy_buffer.txt
```

Comment:
Wouldn't it be cleaner to define another argument that depending on `USE_BUFFER` is either empty or `--entropy-buffer ./util/design/otp_mmap_entropy_buffer.txt` and then always call the scripts with this argument?

Reply:
That's a good idea. I'll change it now. It might also fix the problem with the tabs.

Reply:
> 1. What is missing in your view to also have bazel/dvsim load those buffers? AFAIU bazel/dvsim still generates buffers using Python's `random` on the fly.

This was changed in #18346, `LcStEnc.py` and `OtpMemMap.py` no longer use `entropy_buffer_generator.py`.
Instead, we have added a function `unsecure_generate_from_seed()` to the strong_random class, which generates the buffer and doesn't leave any residual files.
However `topgen.py` at the moment still uses `entropy_buffer_generator.py`. i will update it in a separate PR to be consistent with the other scripts.